README.md

```diff
@@ -13,14 +13,26 @@
 ### Jodi A Hieronymus
 - Project setup, front-end package installation, repository organization
 
-## How To Run
-
 Node version: `v18.12.1`
 
-1 - Run `npm i`
+## To Run UI:
+1 - open terminal, run `cd client`
+
+2 - Run `npm i`
+
+3 - Run `npm start`
+
+4 - Go to `http://localhost:3000/`
+
+5 - Explore OuterWhorld!
+
+## To Run Server:
+1 - open terminal , run `cd server`
+
+2 - Run `npm i`
 
-2 - Run `npm start`
+3 - Run `npm run dev`
 
-3 - Go to `http://localhost:3000/`
+4 - Terminal should say `Server running on port <Number>`
 
-4 - Explore OuterWhorld!
+## It's recommended you have both UI and Server up and Running at the same time!  
```

Comment:
Does it matter which one's running first?

Reply:
I see it doesn't matter now, but do you anticipate it will matter in the future?

Reply:
mmm good question. In the future when we deploy our code I think it'll be up and running constantly so it shouldn't matter. 

If the UI is up and running but the server isn't, then the API requests will fail. So I think "technically" it's better to have the server up and running before the client. But you should have the two running before you begin coding anways, so I don't think it matters that much in the end.